ipfx/nwb_reader.py

```diff
@@ -3,6 +3,15 @@
 import ipfx.stim_features as st
 
 
+def get_long_unit_name(unit):
```

Comment:
This function interprets the nwb data and is better suited for the nwb_reader class. The stim_feaures module includes functions for determining features from traces and should hopefully be independent of any nwb functionality

Reply:
Good point @sgratiy .  @t-b could you address this in the next PR?

Reply:
@dyf @sgratiy I can certainly make this a NwbReader class function. I'll prepare a PR.